### PR TITLE
Change codeowners from syseng to `robhudson`

### PR DESCRIPTION
**Pull request by grahamalama**

### CODE REVIEW

**Comment by grahamalama** [comment]
@robhudson, who else should we add here, if anyone?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1 @@
-# Thanks to @imbstack for the suggestion.
-# see http://code.v.igoro.us/posts/2019/07/codeowners.html
-
-* @mozilla-it/syseng
+* @robhudson
```
Comment (robhudson) [inline .github/CODEOWNERS:1]: I think we can also add `@mozmeao/backend` to include the 3 of us.

Comment (grahamalama) [reply .github/CODEOWNERS:1]: Will do!

Comment (grahamalama) [reply .github/CODEOWNERS:1]: Is that a team in https://github.com/orgs/mozilla-it/? All I was able to find was https://github.com/orgs/mozilla-it/teams/mozmeao. I don't think you can do cross-org codeownership like that, unfortunately

Comment (robhudson) [reply .github/CODEOWNERS:1]: Okay, makes sense. We could use the `@mozmeao` one. I think it would be nice to have a "main" but also a team as fallback.

**Comment by robhudson** [comment]
Updated to use `* @mozilla-it/ctms-reviewers` to more easily manage.

**Review by robhudson — APPROVED** [review]

**Pull request merged**